sandbox/grist/functions/info.py

```diff
@@ -653,20 +655,72 @@ def is_error(value):
       or (isinstance(value, float) and math.isnan(value)))
 
 
+def _replicate_requests_body_args(data=None, json=None):
+  """
+  Replicate some of the behaviour of requests.post, specifically the data and 
+  json args.
+
+  From testing manually with requests 2.28.2, data overrides json if both supplied.
+
+  Returns a tuple of (body, extra_headers)
+  """
+  if data is not None:
+    if isinstance(data, str):
+        body = data
+        extra_headers = {}
+    else:
+        body = urllib.parse.urlencode(data)
+        extra_headers = {
+          "Content-Type": "application/x-www-form-urlencoded",
+        }
+    return body, extra_headers
+
+  if json is not None:
+    if isinstance(json, str):
+        body = json
+    else:
+        body = json_module.dumps(json)
+    extra_headers = {
+      "Content-Type": "application/json",
+    }
+    return body, extra_headers
+
+  return None, {}
+
+
 @unimplemented
 # ^ This excludes this function from autocomplete while in beta
 # and marks it as unimplemented in the docs.
 # It also makes grist-help expect to see the string 'raise NotImplemented' in the function source,
 # which it does now, because of this comment. Removing this comment will currently break the docs.
-def REQUEST(url, params=None, headers=None):
+def REQUEST(url, params=None, headers=None, method=None, data=None, json=None):
   # Makes a GET HTTP request with an API similar to `requests.get`.
   # Actually jumps through hoops internally to make the request asynchronously (usually)
   # while feeling synchronous to the formula writer.
 
+  # REQUEST also has a similar interfact to requests.post, if you change method to be POST.
```

Comment:
"Makes a GET HTTP request" needs updating. Rather than comparing to requests.get or requests.post, this is now similar to `requests.request`